src/components/screens/dashboard/dashboard-screen.tsx

```diff
@@ -1,19 +1,22 @@
-import { useAtom } from "jotai";
-import { authAtom } from "../../../atoms/auth";
+import React from "react";
+import strings from "../../../localization/strings";
+import { useLocale } from "../../../localization/LanguageSwitcher";
 
 /**
  * Dashboard screen component
- * 
  */
 const DashboardScreen = () => {
+  const { locale } = useLocale();
   const [auth] = useAtom(authAtom);
```

Comment:
useAtom is not imported which is causing an error preventing the dashboard-screen rendering